Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:6.17.1
```

Comment:
Can you use node 12 or 14? Node 6 is not maintained anymore.

Reply:
I picked 6.17.1 as it is what the Travis CI build uses. The newer versions of Node are incompatible with vega/voyager -- at least the builds failed for me. It looks like the voyager code uses removed APIs (likely long past deprecated).

The knock/voyager Dockerfile I based this on had used node 9

Reply:
Maybe we should switch to github actions like we did in all other active vega projects.

Reply:
Certainly an option, but I think you'd still need to build with the older version of node until the voyager software is updated. The .travis.yml defines the version of node used, not Travis itself. https://github.com/vega/voyager/blob/master/.travis.yml#L3

Reply:
A quick change to the Dockerfile to use node:latest (presumably 14.4) and it also looks like there are library version incompatibilities. I had tried earlier today to force upgrading all npm libraries to current/latest, but that was unsuccessful too. I hadn't spent the time to dive deeper into the software to determine what versions of each of the dependencies are required. I had really just wanted to get it working, so I fell back to the known working & tested version of 6.17.1.

Reply:
Since node 12 is the oldest supported version, let's hold off on merging this pull request. 

https://nodejs.org/en/about/releases/